lib/npg_warehouse/loader/pacbio/run.pm

```diff
@@ -328,7 +329,8 @@ sub _well_qc_info {
     $qc{'control_read_length_mean'}    = $qc_all->{'Control Read Length Mean'};
     $qc{'local_base_rate'}             = $qc_all->{'Local Base Rate'};
     $qc{'hifi_barcoded_reads'}         = $qc_all->{'Barcoded HiFi Reads'};
-    $qc{'hifi_bases_in_barcoded_reads'} = $qc_all->{'Barcoded HiFi Yield (bp)'};
+    $qc{'hifi_bases_in_barcoded_reads'} = defined $qc_all->{'Barcoded HiFi Yield (bp)'} ?
```

Comment:
Do these numbers need scaling or have they always been Gb in disguise? I'd suggest a comment here to explain why db and Gb are of equal magnitude.

Reply:
New label is Gb but value is actually still in bp ! I've added a comment.